modules/rosa-adding-tuning.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-adding-tuning_{context}"]
+= Adding node tuning to a machine pool
```

Comment:
@andreadecorte this addition should address the tuning on machine pools criteria.